Sources/Extensions/SwiftStdlib/StringProtocolExtensions.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  StringProtocolExtensions.swift
+//  SwifterSwift
+//
+//  Created by Max Härtwig on 11/26/17.
+//  Copyright © 2017 SwifterSwift
+//
+
+import Foundation
+
+public extension StringProtocol where Index == String.Index {
+    
+    /// SwifterSwift: The longest common suffix.
+    ///
+    ///        "Hello world!".commonSuffix(with: "It's cold!") = "ld!"
+    ///
+    public func commonSuffix<T : StringProtocol>(with aString: T, options: String.CompareOptions = []) -> String {
```

Comment:
@Vyax Can you properly document this method by including the parameters and return value?